### ISSUE TITLE
Space between fontawesome icon and category name missing when site is built with --minify flag.

### ISSUE THREAD

**Issue by MarcoIeni** [htr3n/hyde-hyde#99]
If you build the website with `hugo server --minify` the space between fontawesome icon and text in posts is not enough.

similar issue I have found that could be useful:
https://github.com/gcushen/hugo-academic/issues/1219

**Comment by htr3n** [comment]
@MarcoIeni thanks for reporting. I'm not sure whether I caught this one in my setting or not. Do you hav any suggestions or ideas on what we can do with this issue?

**Comment by MarcoIeni** [comment]
No, I have no idea why this happens, I ended up removing the `minify` flag when deploying my website :(

**Comment by gary-kim** [comment]
What I've done on my site is remove the space and use css to make the margin instead. I can make a PR to have that in the theme instead if you'd like.
